### PR TITLE
Add metrics for event rates

### PR DESCRIPTION
**Pull request by rudigiesler**

### CODE REVIEW

**Comment by rudigiesler** [comment]
Ready for review.

File: junebug/tests/test_workers.py [hunk 1]
```diff
@@ -309,6 +309,114 @@ def test_message_rates(self):
         self.assertEqual((yield worker.message_rate.get_messages_per_second(
             'testtransport', 'inbound', 1.0)), 1.0)
 
+    @inlineCallbacks
+    def test_submitted_event_rates(self):
+        '''Ackknowledge events should increase the submitted event rates.'''
```
Comment (hodgestar) [inline junebug/tests/test_workers.py]: s/Ackk/Ack/

File: junebug/workers.py [hunk 2]
```diff
@@ -89,6 +88,24 @@ def store_and_forward_event(self, event):
...
+        yield self._count_event(event)
+
+    def _increment_metric(self, label):
+        return self.message_rate.increment(
+            self.channel_id, label, self.config['metric_window'])
+
+    def _count_event(self, event):
+        if event['event_type'] == 'ack':
+            return self._increment_metric('submitted')
+        if event['event_type'] == 'nack':
```
Comment (hodgestar) [inline junebug/workers.py]: I wonder if we want to use `elif`s here? Might be a bit of a bikeshed.

Comment (rudigiesler) [inline junebug/workers.py]: What would be the advantage? Since we're returning for each branch of the if statements?

Comment (hodgestar) [inline junebug/workers.py]: Oh. Doh. :)

**Comment by rudigiesler** [comment]
Fixed the typo, ready for review again

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**